test/test_context.py

```diff
@@ -256,6 +258,18 @@ def test_cartocontext_write(self):
         # util columns + new column of type number
         self.assertDictEqual(cols['fields'], expected_schema)
 
+        # test properly encoding
+        df = pd.DataFrame({'vals':[1,2],'strings':['a','ô']})
+        cc.write(df, self.test_write_table, overwrite=True)
+
+        # check if table exists
+        resp = self.sql_client.send('''
+            SELECT *
+            FROM {table}
+            LIMIT 0
+            '''.format(table=self.test_write_table))
+        self.assertIsNotNone(resp)
```

Comment:
One thing you could test here is if the character sent is the same that's in the database